elastic/kibana
# [RCA] Start investigation from alert details page

Resolves https://github.com/elastic/kibana/issues/190320 and https://github.com/elastic/kibana/issues/190396

- Start investigation from Custom threshold alert details page
- Go to ongoing investigation instead of creating new one if one already exists
- Initial investigation status is set as `ongoing`
- Investigation origin is set as `alert`

"Start investigation" is hidden for other alert types and when investigate plugin is disabled.

### Testing
- Add the following in `kibana.dev.yml`
```
xpack.investigate.enabled: true
xpack.investigateApp.enabled: true
```
- Create Custom threshold rule
- Open Custom threshold alert details page
- Click on "Start investigation"
- Verify that a new saved object is created for the investigation

https://github.com/user-attachments/assets/6dfe8a5f-287b-4cc5-92ae-e4c315c7420b
